core/lib/types/src/storage_writes_deduplicator.rs

```diff
@@ -11,6 +12,8 @@ pub struct ModifiedSlot {
     pub value: U256,
     /// Index (in L1 batch) of the transaction that lastly modified the slot.
     pub tx_index: u16,
+    /// Size of pubdata update in bytes
+    pub size: usize,
```

Comment:
nit: I'd prefer to change field name to `compressed_size` as it is more explicit